app/services/creators/employments_creator.rb

```diff
@@ -47,7 +57,7 @@ def create_payments(employment)
     end
 
     def assessment
-      @assessment ||= Assessment.find_by(id: assessment_id) || (raise CreationError, ["No such assessment id"])
+      @assessment ||= Assessment.find_by(id: @assessment_id) || (raise CreationError, ["No such assessment id"])
```

Comment:
**not a blocker**, This is called at line 30 to raise errors but is not subsequently used. It could be used in place of @assesment_id (use assessment.id)